database-service/src/main/java/com/example/databaseservice/model/HelloWorld.java

```diff
@@ -0,0 +1,47 @@
+package com.example.databaseservice.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "HELLO_WORLD")
+
+public class HelloWorld {
+
+    @Id
+    @GeneratedValue
+
+    private Long id;
+
+    private String message;
+
+    public HelloWorld() {
+    }
+
+    public HelloWorld(String message) {
+        this.message = message;
+    }
+
+    public String getMessage() {
+        return message;
+    }
+
+    public void setMessage(String message) {
+        this.message = message;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    @Override
+    public String toString() {
+        return "HelloWorld{" +

```

Comment:
Jakie są lepsze sposoby na budowanie zawartości metody "toString"?

Reply:
Użycie StringBiulder

StringBuilder stringBuilder = new StringBuilder();
        stringBuilder.append("id= "+this.id+"message= "+this.message+"");
        return stringBuilder.toString();

Reply:
Akurat takie użycie StringBuilder'a nic nie poprawi. Dlaczego?

Reply:
Bo zastosowaliśmy konkatenację. Powinno być: stringBuilder.append("id=").append(this.id).append("message= ").append(this.message);